docs/packages/patches.json

```
@@ -0,0 +1,10 @@
+
+{
+  "patches" : [
+        {
+          "file" : "Thrust/cub_odr.diff",
+          "issue" : "cub kernel dispatch ODR [https://github.com/NVIDIA/cub/issues/545]",
+          "fixed_in" : ""
+        }
+      ]
+}
```

Format this to something like:
```suggestion
{
  "patches" : [
    {
      "file" : "Thrust/cub_odr.diff",
      "issue" : "cub kernel dispatch ODR [https://github.com/NVIDIA/cub/issues/545]",
      "fixed_in" : ""
    }
  ]
}
```